Workshop.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You probably shouldn't commit this file. The Workshop test creates this files in the root and never deletes it, which is wrong.